Aliucord/src/main/java/com/aliucord/coreplugins/plugindownloader/PluginDownloader.kt

```diff
@@ -71,6 +73,18 @@ internal class PluginDownloader : Plugin() {
                                 }
                             }
                         }
+
+                        if (msg.hasAttachments()) {
```

Comment:
i don't like this also being in Plugin links and plugin support because we ban sending plugin zips in support because thats dangerous as some random may send malicious builds of plugins, so this should only happen in plugin dev and no other channel. You can just make this its own when case since noone is gonna send a github link in plugin dev